public/scss/components/Map.module.scss

```
@@ -154,6 +154,11 @@
     padding: 8px 16px;
   }
 
+  &_mapPinPointIcon
+  {
+    @include absolute(50%,auto,auto,50%);
```

```suggestion
    @include absolute(50%, auto, auto, 50%);
    
    transform: translate3d(-50%, -50%, 0); // tambahin ini biar icon map pin point-nya jadi ke tengah
```
tiap parameter mixin, pisahin pake spasi